contracts/model/CredmarkModel.sol

```diff
@@ -19,7 +18,7 @@ contract CredmarkModel is ERC721, Pausable, ERC721Enumerable, AccessControl {
     mapping(uint256 => uint256) public slugHashes;
     mapping(uint256 => uint256) private slugTokens;
 
-    event LogCredmarkModelMinted(uint256 slugHash);
+    event NFTMinted(uint256 tokenId);
```

Comment:
Don't remove `slugHash`. Also make the `tokenId` param indexed.

Reply:
done